css/layouts/responsive.css

```diff
@@ -80,6 +80,58 @@
   }
 }
 
+/* Footer ABN and Tagline */
+@media all and (min-width: 580px) {
+  /* The footer abn and tagline */
+  .region-footer .block {
+    width: 50%;
+  }
+  .region-footer .menu::after {
+    position: relative;
+    float: left;
+    clear: both;
+    display: block;
+    content: "ABN 63 384 330 717";
```

Comment:
I'd be tempted to put the ABN in a template file. Consider looking at region--footer.tpl.php for this. You will be able to copy region--footer.tpl.php from the zen theme into this theme as a starter.